gpu_ocean/SWESimulators/Common.py

```diff
@@ -212,6 +212,10 @@ def __exit__(self, *args):
     def getPrintString(self, x):
         elapsed =  time.time() - self.start
         
+        if (x == 0):
+            self.print_string = ProgressPrinter.formatString(0, 0, np.nan)
```

Comment:
Bedre å bruke None her

Reply:
Det ble verre å bruke None. Mange operasjonene som brukes er veldefinerte for NaN, men ikke for None (+, max etc). Fortsetter derfor med np.nan.